mobiusTutorial.md

```diff
@@ -0,0 +1,299 @@
+# Sending transactions with Mobius
+
+## Environment
+
+- Truffle:
+```bash
+$ truffle version
+Truffle v4.1.0 (core: 4.1.0)
+Solidity v0.4.19 (solc-js)
+```
+
+- Geth:
+```bash
+$ geth version
+Geth
+Version: 1.8.1-stable
+Git Commit: 1e67410e88d2685bc54611a7c9f75c327b553ccc
+```
+
+- Npm:
+```bash
+$ npm --version
+5.6.0
+```
+
+- Node:
+```bash
+$ node --version
+v9.5.0
+```
+
+## How to use Mobius ?
+
+### Assumptions
+
+1. The `Mixer` has a `RING_SIZE` equal to 1 (see: https://github.com/clearmatics/mobius/blob/master/contracts/LinkableRing.sol#L80).
```

Comment:
Ring size is 4.

Reply:
This is an hardcoded value. I assume that the ring size is 1 to make the tuto easier and shorter. The file has to be edited before the contracts are deployed. See:
```
Assumptions

    The Mixer has a RING_SIZE equal to 1 (see: https://github.com/clearmatics/mobius/blob/master/contracts/LinkableRing.sol#L80). If you want to follow the rest of the tutorial, please clone this repository and modify the LinkableRing contract accordingly:

export $WORKDIR = ~/Path/To/Your/Working/Directory
git clone https://github.com/clearmatics/mobius.git $WORKDIR
cd $WORKDIR/mobius
[YourFavoriteTextEditor] contracts/LinkableRing.sol
[Change line 80: uint256 public constant RING_SIZE = 4; INTO uint256 public constant RING_SIZE = 1;]
[Save and exit your editor]
```
The purpose here is not to simulate a real flow but just to show how things work together. Then people are free to change the RING_SIZE as they want to play around.